proto/ibc-attestor/attestation.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package ibc_attestor;
+
+// Attestation is a single attestation from a given block height for the requested data.
+message Attestation {
+  // The height of the attestation
+  uint64 height = 1;
```

Comment:
I'm a bit unsure if the height should be in the attestation itself, or a separate field in the response. Would like feedback on that, if you have any. If unsure, we can change that later, too.